### DISCUSSION TITLE
Streams play faster

### DISCUSSION THREAD

**Discussion by JA16122000** [mrlt8/docker-wyze-bridge#540] [Q&A]
Hi there!

I'm wondering, why my both streams, are playing faster then normal.
te timelaps under both streams, are playing really fast (1 second is in my opinion 500ms)

**Comment by mrlt8** [comment]
Could you post the camera models and what stream (RTSP/HLS) you're using?

We've had a few issues with the stream playing slower, but never had an issue with it playing faster.

**Reply by JA16122000** [reply]
Well, i'm using the Wyze Cam V3 (both)
Those with the "beta firmware" (rtsp)

Now it is mainly one of the two camera's, thats playing much faster then the other

**Reply by mrlt8** [reply]
Is this a temporary speed up? I'm guessing it eventually catches up to realtime?

**Reply by JA16122000** [reply]
It's building up, so i think.
But it is only one camera.
On the wyze app it self, it seems fine, but on the web, it seems fast playing.
I will reboot the Raspberry Pi, to see if that is helping.

**Reply by JA16122000** [reply]
Rebooting the Raspberry Pi did not help.

**Reply by mrlt8** [reply]
Is this the HLS stream that's playing faster? could you try the RTSP/RTMP to see if you get a similar result?

Also, does the playback speed change when night vision is on?

**Reply by JA16122000** [reply]
The RTSP stream (URL thats coming from the bridge) does not work.
Also the same for the RTMP stream.

HLS works fine, but very fast.

Night vision or daylight, doesn't change the speed

Maybe i can send you the url from my site? (In private, if i know how)

**Reply by mrlt8** [reply]
Could you try opening the streams in a media player like VLC and see if it still has the speed issue?

**Reply by JA16122000** [reply]
The streams (RTSP and RTMP) given by the bridge, didn't work in VLC
The RTSP stream given by Wyze itself, works fine on VLC

**Reply by mrlt8** [reply]
Hmm I'm not sure why the other streams aren't working... Do you have the appropriate ports exposed for the container?

Could you also see if the HLS stream is having issues in VLC with `http://<bridge>:8888/camera-nickname/stream.m3u8`

**Reply by JA16122000** [reply]
That seems working. But also fast playing. Much to fast

**Reply by JA16122000** [reply]
port 1935 and 8554 was i forgot to forward :P
that is now also working.
Yesterday, i rebooted the camera, that was playing way to fast.
Now it seems to be fine.
I'll take the next week, to look off it will speedup.
So lets pauze this issue and lets see what it bringing us this week!
